src/authorization.ts

```diff
@@ -103,4 +125,4 @@ export interface Authorization {
      * @returns Search filter
      */
     getSearchFilterBasedOnIdentity(request: GetSearchFilterBasedOnIdentityRequest): Promise<SearchFilter[]>;
-}
+}
```

Comment:
Minor linting required -- locally you can run `yarn run release` to test against the given linter & unit tests

Reply:
Fixed. (my IDE was removing newlines at end of file. Changed the config.)